### DISCUSSION TITLE
TypeError: compiler.plugin is not a function at ReactLoadablePlugin.apply

### DISCUSSION THREAD

**Discussion by sushmitg** [webpack/webpack#16715] [Q&A]
I am getting the following error with react-loadable/webpack after upgrading to webpack 5:

![image](https://user-images.githubusercontent.com/22211683/215751140-56bda56c-ddbb-488c-acfb-f7ad91828378.png)

"react-loadable": "^5.5.0",
"webpack": "^5.75.0",
"webpack-cli": "^5.0.1",

code:
```
const { ReactLoadablePlugin } = require('react-loadable/webpack');
  ...

plugins: [
    new ReactLoadablePlugin({
      filename: path.resolve('./build/client') + `/react-loadable-${config.name}.json`,
    }),
```

**Comment by alexander-akait** [comment]
Please provide full configuration, thank you
